app/services/service_provider_updater.rb

```diff
@@ -1,9 +1,10 @@
 class ServiceProviderUpdater
   PROTECTED_ATTRIBUTES = [
-    :id,
+    :approved,
```

Comment:
Gotcha. I'm assuming the `approved` flag can only be set by admins on the dashboard, and that we prevent people from manually changing the form such that is submits the approved attribute?